app/routes/admin/content/index.js

```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('store').queryRecord('setting', {});
+  },
+  actions: {
+    willTransition() {
+      this.get('controller.model').rollbackAttributes();
```

Comment:
@niranjan94 Of course sorry, was testing something and pasted it everywhere.  I will change it